src/data/Data.js

```diff
@@ -577,9 +577,10 @@ export const WorkspaceData = _.flow(
     const region = isAzureWorkspace ? storageDetails.azureContainerRegion : storageDetails.googleBucketLocation;
 
     const wdsDataTableProvider = useMemo(() => {
-      const proxyUrl = !!wdsProxyUrl && wdsProxyUrl.state;
+      const app = wdsApp?.state;
+      const proxyUrl = app?.proxyUrl;
```

Comment:
oopsie 1: `proxyUrl` isn't actually a field on app here, so this would always be undefined